contents/blog/how-to-do-user-segmentation.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2021-10-28
+title: How to do user segmentation
```

Comment:
Are the names of these two pieces of content too similar?

Reply:
I'll change the blog to 'What is User Segmentation?', I think. We just had a discussion in marketing sprint about how to distinguish tutorials, blogs, etc.